apps/site/assets/css/_trip-plan-results.scss

```diff
@@ -1,27 +1,34 @@
 .m-trip-plan-results__itinerary-container {
   margin-bottom: $base-spacing * 1.5;
 
+  h2 {
+    font-size: 1.2em;
+    margin-top: 0;
+  }
+
   .c-expandable-block__header {
```

Comment:
Is `c-expandable-block__header` maybe not so relevant now that this is more of a block above an accordian? I wonder if it makes sense to rename it—you'd have to reproduce some of the styling, but you'd also no longer have to override other styles (like the cursor).